odl/tomo/backends/scikit_radon.py

```diff
@@ -0,0 +1,106 @@
+
+def make_scikit_theta(geometry):
+    """ Calculate angles in degrees with ODL scikit conventions. """
+    return np.asarray(geometry.motion_grid).squeeze() * 180.0 / np.pi - 90
+
+def make_scikit_range(volume_shape, geometry, range):
+    """ Create a range adapted to the scikit radon geometry. """
+    # TODO: this is wrong min/max
+    padded_size = int(np.ceil(volume_shape[0] * np.sqrt(2)))
+    scikit_detector_part = uniform_partition(range.partition.min()[1],
+                                             range.partition.max()[1],
+                                             padded_size)
+
+    scikit_range_part = geometry.motion_partition.insert(1,
+                                                         scikit_detector_part)
+
+    scikit_range = uniform_discr_frompartition(scikit_range_part,
+                                               interp=range.interp,
+                                               dtype=range.dtype)
+
+    return scikit_range
+
+
+def scikit_radon_forward(volume, geometry, range, out=None):
+    """ Calculate forward projection using scikit
+
+    Parameters
+    ----------
+    volume : DiscreteLpVector
+        The volume to project
+    geometry : Geometry
+        The projection geometry to use
+    range : DiscreteLp
+        range of this projection (sinogram space)
+    out : range element, optional
+        An element in range that the result should be written to
+
+    Returns
+    -------
+    sinogram : range element
+        The sinogram given by the projection
+    """
+    theta = make_scikit_theta(geometry)
+    scikit_range = make_scikit_range(volume.shape, geometry, range)
+
+    sinogram = scikit_range.element(radon(volume.asarray(), theta=theta).T)
+
+    if out is None:
+        out = range.element()
+    out.sampling(sinogram.interpolation)
+
+    scale = volume.space.cell_sides[0]
+
+    out *= scale
+
+    return out
+
+
+def scikit_radon_back_projector(sinogram, geometry, range, out=None):
+    """ Calculate forward projection using scikit
+
+    Parameters
+    ----------
+    sinogram : DiscreteLpVector
+        The sinogram (projections) to backproject
+    geometry : Geometry
+        The projection geometry to use
+    range : DiscreteLp
+        range of this projection (volume space)
+    out : range element, optional
+        An element in range that the result should be written to
+
+    Returns
+    -------
+    sinogram : range element
+        The sinogram given by the projection
+    """
+    theta = make_scikit_theta(geometry)
+    scikit_range = make_scikit_range(range.shape, geometry, sinogram.space)
+
+    scikit_sinogram = scikit_range.element()
+    scikit_sinogram.sampling(sinogram.interpolation)
+
+    if out is None:
+        out = range.element()
+
+    out[:] = iradon(scikit_sinogram.asarray().T, theta,
+                    output_size=range.shape[0], filter=None)
+
+    # Empirical value, gives correct scaling
+    scale = 4.0
```

Comment:
No idea why, as comment above sais, its empirical and gives the correct scaling (same as ASTRA)